### ISSUE TITLE
keep icon colors in tabs

### ISSUE THREAD

**Issue by ilmam** [atom-material/atom-material-ui#349]
I use "file-icons" to display colored icons in atom tabs. After switching to atom-material-ui, and setting tainted tab bar color. I lost the icons colors. This is important to locate the files easily.
Can I hack this icon colors in material-ui ?

**Comment by silvestreh** [comment]
I [overrode the color](https://github.com/atom-material/atom-material-ui/blob/master/styles/tabs.less#L311) when using tinted tabs for the simple reason that if you have a green icon with a green background you wouldn't see it. I doubt that this can be reverted by using your user stylesheets, but it doesn't hurt to try.

**Comment by eminos** [comment]
It seems the icon colors are lost on 2.0, even without Tinted tab bar.

**Comment by doryphores** [comment]
@silvestreh This line [here](https://github.com/atom-material/atom-material-ui/blob/2.0/styles/tabs.less#L191) overrides tab icon colors regardless of tinted tab use.

**Comment by sbywater-rp** [comment]
This bugs me a lot too. @doryphores link is broken, [here](https://github.com/atom-material/atom-material-ui/blob/master/styles/tabs.less#L191) is the code in master that overrides the color. 
Making this color override optional would be much appreciated.

**Comment by mrleblanc101** [comment]
I've added this to my stylesheet in the meantime...
```css
.amu-tinted-tab-bar atom-pane .tab-bar .tab .title[data-path]::before {
    color: inherit !important;
}

atom-pane .tab-bar .tab .title[data-path]::before {
    color: inherit !important;
}
```
I hope this is getting a fix soon, I don't like that atom-material-ui try to dictate how other packages should behave. My custom theme color has enough contrast with the yellow and green using by `VCS coloring` option in atom `tab` and `file-icons`.
<img width="612" alt="capture d ecran 2017-10-01 a 17 10 51" src="https://user-images.githubusercontent.com/17392251/31059007-841353be-a6cb-11e7-8298-2dc7153f523b.png">

**Comment by Alhadis** [comment]
Hey guys, maintainer of `file-icons` here. I've submitted a [pull-request](https://github.com/atom-material/atom-material-ui/issues/349) to address this. However, it's dependent on a [change](https://github.com/file-icons/atom/commit/5e025c2614d88405dbe206a32d4e51c8c4c3986e) I had to make to the package, so it therefore won't take hold until the next release. I'll endeavour to get one cut shortly after the PR's made it upstream.
